### PR TITLE
Add prediction examples

### PR DESCRIPTION
**Pull request by adambrinkman**
Examples of the prediction format

### CODE REVIEW

File: docs-src/docs/dataset-refinement/prediction-format.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+---
+
+Predictions use the OpenLabel format/schema. This is the same format as the one used
+for [uploading pre-annotations](../kognic-io/pre_annotations/#openlabel-support). General information about the
+OpenLabel format can be found in [here](../openlabel/openlabel-format).
+
+## Supported prediction features
+
+:::note
+Only one type of feature per prediction is supported.
```
Comment (TheoBerglin) [inline docs-src/docs/dataset-refinement/prediction-format.md:12]: I didn't understand what a feature was when I read this, maybe change it to "Only one type of feature, for example cuboid in 3D, per prediction is supported" or something similar

File: docs-src/docs/dataset-refinement/prediction-format.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+:::
+
+The current API for uploading predictions supports the following features:
+
+| Name         | OpenLABEL field | Description        |
+|--------------|-----------------|--------------------|
+| Cuboid       | `cuboid`        | Cuboid in 3D       |
+| Bounding box | `bbox`          | Bounding box in 2D |
+
+Note that all geometries should be specified under frames rather than in the root of the pre-annotation. The rotation of
```
Comment (TheoBerglin) [inline docs-src/docs/dataset-refinement/prediction-format.md:22]: Mix geometries and features. Maybe geometries is the best?

File: docs-src/docs/dataset-refinement/prediction-format.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+resolved automatically if it is left empty.
+
+Existence confidence can be provided by specifying an attribute called `confidence`. It is not required and will be set
+to 1.0 if it is left empty. If provided, it must be defined as a numeric value between 0.0 and 1.0. Existence confidence
+is set to 0.85 in the examples below.
+
+The `camera_id` and `lidar_id` in the examples below must match the id of the sensor in already existing judgements.
+
+The `object_data.type` will show up as the class name in the tool.
+
```
Comment (TheoBerglin) [inline docs-src/docs/dataset-refinement/prediction-format.md:38]: I really like these, is there any way we can highlight the important parts? For example, we could make the part "`frame_properties` is not required" bold

**Review by TheoBerglin — APPROVED** [review]
Amazing work with this. Documenting stuff is really difficult. Would be interesting if the "chat with a guideline feature" could be extended to "chat with our documentation"

File: docs-src/docs/dataset-refinement/prediction-format.md (hunk 1 above)
Comment (adambrinkman) [reply docs-src/docs/dataset-refinement/prediction-format.md:12]: Definitely. Agree!

File: docs-src/docs/dataset-refinement/prediction-format.md (hunk 2 above)
Comment (adambrinkman) [reply docs-src/docs/dataset-refinement/prediction-format.md:22]: Good point, will change to geometries

File: docs-src/docs/dataset-refinement/prediction-format.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+center coordinates of the bounding box. The `width` and `height` are the width and height of the bounding box. The `x`
+and `y` coordinates are relative to the upper left corner of the image.
+
+```json
+{
+  "openlabel": {
+    "frames": {
+      "0": {
+        "objects": {
+          "1232b4f4-e3ca-446a-91cb-d8d403703df7": {
```
Comment (andreas-lindholm) [inline docs-src/docs/dataset-refinement/prediction-format.md:52]: Maybe some explanation on the requirements here? I guess any UUID identifier they generate themselves will suffice? Maybe it goes without saying that they have to be unique in each file (which I think is the only thing we require, right?)

**Pull request merged**